src/caselawclient/models/utilities/aws.py

```diff
@@ -13,9 +13,18 @@
 from mypy_boto3_sns.type_defs import MessageAttributeValueTypeDef
 from typing_extensions import NotRequired
 
+if TYPE_CHECKING:
+    from caselawclient.models.documents import DocumentURIString
+else:
+    DocumentURIString = None
+
 env = environ.Env()
 
 
+class S3PrefixString(str):
```

Comment:
Is it worth adding some validation here (see eg https://github.com/nationalarchives/ds-caselaw-custom-api-client/blob/87799a998ddea62fe7f0c9fce12ba00f25b798c3/src/caselawclient/models/documents/__init__.py#L58) to prove that the string ends with a `/`?